one_fm/tasks/execute.py

```diff
@@ -0,0 +1,11 @@
+import frappe
+from frappe import _
+from frappe import enqueue
+
+
+
+def daily():
```

Comment:
You can call the method directly in the daily scheduler in hooks instead of adding a new method for tasks.

Reply:
No, this queues it up before execution using enqueue

Reply:
What I mean us you can call `due_purchase_order_payment_terms` directly in hooks instead of enqueuing it manually.

Reply:
yea but will extend the line, if tasks are grouped and queued, the hooks will be much cleaner.
def daily():
   task1
   task2
   task3
   ....

Reply:
Sure.